internal/parse/parts.go

```diff
@@ -52,8 +55,16 @@ func (p *OutputPart) String() string {
 	return fmt.Sprintf("OutputPart[%+v %+v]", p.Source, p.Target)
 }
 
-func (p *OutputPart) ToSQL() string {
-	return ""
+func (p *OutputPart) ToSQL(cs []string) string {
+	var out bytes.Buffer
+	out.WriteString(" ")
+	for i, c := range cs {
+		if i > 0 {
```

Comment:
Is this method meant to return a string as "`cs[0], cs[1], ... , cs[len-1]`" ? (i.e. a "`, `"-separated sequence of words). If so, maybe we could then do `if i != len(s)-1` instead of `if i>0`, and move the `if` block after `out.WriteString(c)` (so for `i=0,...len-2`), so that no comma is added after the last element. Is the initial `out.WriteString(" ")` there to prevent an empty string output if cs is empty? If so, would it make sense to do `if (len(s)==0)` right before `out.WriteString(" ")`